lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+    
+  
+  def move(input = nil)
+    input = gets.strip
+    input
```

Comment:
Because of Ruby's implicit return, `input` is unnecessary in this method. It could be simplified to
```
def move(input = nil)
  input = gets.string
end
```
But since `input` isn't being used elsewhere in this method, it could be further simplified to 
```
def move(input = nil)
  gets.strip
end
```